evm_loader/utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Permissions management scripts
+
+Scripts mint_permission_token.sh mint_many.sh are suited to operate with client's and contract's permission token balances - to grant or deprive permissions.
+
+## Requirements
+
+To use this scripts one should have next CLI utilities installed:
+    - solana CLI interface
+    - neon-cli 
+    - spl-token program
+
+Non-zero SOL balance is needed on selected Solana network on default account ~/.config/solana/id.json
+Also, user should have correct Mint-authority JSON file
```

Comment:
The same, we don't have and I believe never had any mention about this file
Alexander N. mentioned some folder (?)

Probably, here we need to mention where this fail could be withdrawn as a template or how to create it

Reply:
This file created automatically when installing solana it is not related to Neon-EVM directly